Core/include/Acts/Seeding/EstimateTrackParamsFromSeed.hpp

```diff
@@ -237,7 +237,7 @@ std::optional<BoundVector> estimateTrackParamsFromSeed(
   ActsScalar rn = local2.x() * local2.x() + local2.y() * local2.y();
   // The (1/tanTheta) of momentum in the new frame,
   ActsScalar invTanTheta =
-      local2.z() * std::sqrt(1. / rn) / (1. + rho * rho * rn);
+      local2.z() * std::sqrt(1. / rn) / (1. + rho * rho * rn / 24.f);
```

Comment:
Since 1/24.f is a const what about defining it as a static const value so that you avoid doing a division all the time?

Reply:
Makes sense - I made 1./24 a static const double